src/v1/container_analysis_client.ts

```diff
@@ -253,68 +308,70 @@ class ContainerAnalysisClient {
    *
    * @param {Object} request
    *   The request object that will be sent.
-   * @param {string} request.resource
-   *   REQUIRED: The resource for which the policy is being specified.
-   *   See the operation documentation for the appropriate value for this field.
-   * @param {Object} request.policy
-   *   REQUIRED: The complete policy to be applied to the `resource`. The size of
-   *   the policy is limited to a few 10s of KB. An empty policy is a
-   *   valid policy but certain Cloud Platform services (such as Projects)
-   *   might reject them.
-   *
-   *   This object should have the same structure as [Policy]{@link google.iam.v1.Policy}
-   * @param {Object} [options]
-   *   Optional parameters. You can override the default settings for this call, e.g, timeout,
-   *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/interfaces/CallOptions.html} for the details.
-   * @param {function(?Error, ?Object)} [callback]
-   *   The function which will be called with the result of the API call.
-   *
-   *   The second parameter to the callback is an object representing [Policy]{@link google.iam.v1.Policy}.
+   * @param {object} [options]
+   *   Call options. See {@link https://googleapis.dev/nodejs/google-gax/latest/interfaces/CallOptions.html|CallOptions} for more details.
```

Comment:
Interesting: I wonder why we list those extra comments for each field in the request object. That's OK for now, but may I ask you @xiaozhenliu-gg5 to create an issue to figure out if we can have those comments back if it's not too hard? They look useful.

Reply:
sure, created an issue, will take a look